openmdao/core/problem.py

```diff
@@ -2543,6 +2544,33 @@ def get_outputs_dir(self, *subdirs, mkdir=True):
         """
         return _get_outputs_dir(self, *subdirs, mkdir=mkdir)
 
+    def get_coloring_dir(self, mode, mkdir=False):
+        """
+        Get the path to the directory for the coloring files.
+
+        Parameters
+        ----------
+        mode : str
+            Must be one of 'input' or 'output'. A problem will always write its coloring files to
+            its standard output directory in `{prob_name}_out/coloring_files`, but input coloring
+            files to be loaded may be read from a different directory specifed by the problem's
+            `coloring_dir` option.
+        mkdir : bool
+            If True, attempt to create this directory if it does not exist.
+
+        Returns
+        -------
+        pathlib.Path
+            The path to the directory where reports should be written.
+        """
+        if mode == 'input':
+            return pathlib.Path(self.options['coloring_dir'])
```

Comment:
It doesn't looks like the 'input' mode is used in the code or covered in the tests.